tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/BMapValueTest.java

```diff
@@ -46,27 +43,27 @@
 
     private CompileResult programFile;
 
-    @Test
-    public void testStandardJavaMap() {
-        // Standard Map
-        Map<String, Integer> map = new HashMap<String, Integer>();
-        map.put("Chanaka", 1);
-        map.put("Udaya", 2);
-        map.put("Chanaka", 1);
-        assertEquals((int) map.get("Chanaka"), 1);
-
-        for (int i = 0; i < 100; i++) {
-            map.put(String.valueOf(i), i);
-        }
-        assertEquals(map.size(), 102);
-        assertEquals((int) map.get("51"), 51);
-
-        map.remove("Chanaka");
-        assertEquals(map.size(), 101);
-
-        map.remove("WSO2");
-        assertEquals(map.size(), 101);
-    }
+//    @Test
```

Comment:
Why are we commenting this out?

Reply:
Do we need this test always running?
This is testing the standard java map.

Reply:
:smiley: Shall we remove this altogether? Instead of commenting it out?

Reply:
Done